awx/main/models/inventory.py

```diff
@@ -1,58 +1,45 @@
 # Copyright (c) 2015 Ansible, Inc.
 # All Rights Reserved.
 
+import copy
+
 # Python
 import datetime
-import time
 import logging
-import re
-import copy
 import os.path
+import re
+import time
 from urllib.parse import urljoin
 
+from ansible_base.lib.utils.models import prevent_search
```

Comment:
Let's keep the order of the imports as it was. Making formatting changes in functional PRs extends their scope to unlimited, meaning that those things that's gone wrong with formatting will now block the entire PR even if the original functional change is complete.
Besides, it's customary to have generic frameworks before libraries.

Reply:
Another side effect of mixing formatting and functional changes is that the reviewers would not know which lines are just formatting and which ones contain functional changes, meaning that they have to inspect every single line of the change, which increases the cognitive load infinitely. This is why it's usually forbidden to do formatting and useful changes in the same PR.

Reply:
My fault entirely- I had been working on another repo in which the flow was to always run Black and Isort before committing, and I erroneously did that here. Thank you for catching that! Those changes have been reverted- and you are 100% right, it is confusing!